src/redempt/redlib/enchants/CustomEnchant.java

```diff
@@ -335,5 +335,9 @@ public String getLore(int level) {
 		}
 		return getDisplayName() + " " + toRomanNumerals(level);
 	}
-	
+
+	public ItemStack increaseLevel(ItemStack itemStack, CustomEnchant customEnchant, int amount) {
+		if (customEnchant.getLevel(itemStack) + amount > customEnchant.getMaxLevel()) return itemStack;
```

Comment:
What if they pass in 10 and the max level is 5, but the item is unenchanted? The returned item would also be unenchanted. You should use Math.min here to ensure that the level doesn't exceed the max level of the item, but will still max it out instead of doing nothing if it goes over.